docs/10-events-and-listeners.md

```diff
@@ -58,7 +58,7 @@ class PostController extends Controller
 }
 ```
 
-### Testing we're emitting the event
+### Testing We're Emitting the Event
```

Comment:
```suggestion
### Testing the Event was Emitted
```

Reply:
Good catch - the 'we are' stood out to me during the change too - it's not common to use we/you/I in the headings.